students/andrew_garcia/lesson02_Grid/Grid_Printer_1.py

```diff
@@ -0,0 +1,30 @@
+'''
+Andrew Garcia
+Grid Printer 1
+6/2/19
+'''
+
+def print_grid():
+    def print_horizontal():
+        print("+ - - - - + - - - - +")
+
+    def print_vertical():
+        print("|         |         |")
+
+    def print_multiple_vertical():
+        print_vertical()
+        print_vertical()
+        print_vertical()
+        print_vertical()
+
+    def entire_grid():
+        print_horizontal()
+        print_multiple_vertical()
+        print_horizontal()
+        print_multiple_vertical()
+        print_horizontal()
+
+    entire_grid()
```

Comment:
there is no need for these to be nested function; better to define them individually